src/obr/signac_wrapper/operations.py

```diff
@@ -575,6 +575,11 @@ def run_cmd_builder(job: Job, cmd_format: str, args: dict) -> str:
         "timestamp": timestamp,
         "np": get_number_of_procs(job),
     }
+    preflight = os.environ.get("OBR_PREFLIGHT")
+    if preflight:
+        preflight_cmd = f"{preflight} > {job.path}/case/preflight_{timestamp}.log && "
+        cmd_format = preflight_cmd + cmd_format
+
     return cmd_format.format(**cli_args) + "|| true"
```

Comment:
What is the `"|| true"` for?

Reply:
Otherwise, if one job fails OBR exits and all other jobs are not executed. I'll add a comment.